po/ar.po

```diff
@@ -747,222 +748,227 @@ msgstr "المتغيرات"
 
 #: src/SUMMARY.md
 msgid "Building for Android: Rust"
-msgstr ""
+msgstr "البناء لأندرويد: Rust (Building for Android: Rust)"
 
 #: src/SUMMARY.md
 msgid "With Java"
-msgstr ""
+msgstr "مع Java (With Java)"
 
 #: src/SUMMARY.md src/exercises/android/morning.md
 #: src/exercises/bare-metal/morning.md src/exercises/bare-metal/afternoon.md
 #: src/concurrency/sync-exercises.md src/concurrency/async-exercises.md
 msgid "Exercises"
-msgstr ""
+msgstr "التمارين"
 
 #: src/SUMMARY.md
 msgid "Chromium"
-msgstr ""
+msgstr "كروميوم (Chromium)"
 
 #: src/SUMMARY.md src/chromium/cargo.md
 msgid "Comparing Chromium and Cargo Ecosystems"
 msgstr ""
+"مقارنة بين بيئات Chromium و Cargo (Comparing Chromium and Cargo Ecosystems)"
 
 #: src/SUMMARY.md
 msgid "Policy"
-msgstr ""
+msgstr "سياسة الاستخدام (Policy)"
 
 #: src/SUMMARY.md
 msgid "Unsafe Code"
-msgstr ""
+msgstr "الكود غير الآمن (Unsafe Code)"
 
 #: src/SUMMARY.md src/chromium/build-rules/depending.md
 msgid "Depending on Rust Code from Chromium C++"
 msgstr ""
+"الاعتماد على كود Rust من Chromium C++ (Depending on Rust Code from Chromium "
+"C++)"
 
 #: src/SUMMARY.md src/chromium/build-rules/vscode.md
 msgid "Visual Studio Code"
-msgstr ""
+msgstr "فيجوال ستوديو كود (Visual Studio Code)"
 
 #: src/SUMMARY.md src/exercises/chromium/third-party.md
 msgid "Exercise"
-msgstr ""
+msgstr "تمرين (Exercise)"
 
 #: src/SUMMARY.md src/chromium/testing/rust-gtest-interop.md
 msgid "`rust_gtest_interop` Library"
-msgstr ""
+msgstr "مكتبة `rust_gtest_interop`"
 
 #: src/SUMMARY.md src/chromium/testing/build-gn.md
 msgid "GN Rules for Rust Tests"
-msgstr ""
+msgstr "قواعد GN لاختبارات Rust (GN Rules for Rust Tests)"
 
 #: src/SUMMARY.md src/chromium/testing/chromium-import-macro.md
 msgid "`chromium::import!` Macro"
-msgstr ""
+msgstr "ماكرو Macro `chromium::import!`"
 
 #: src/SUMMARY.md src/chromium/interoperability-with-cpp.md
 msgid "Interoperability with C++"
-msgstr ""
+msgstr "التشغيل البيني مع C++ (Interoperability with C++)"
 
 #: src/SUMMARY.md src/chromium/interoperability-with-cpp/example-bindings.md
 msgid "Example Bindings"
-msgstr ""
+msgstr "أمثلة على الربط (Example Bindings)"
 
 #: src/SUMMARY.md src/chromium/interoperability-with-cpp/limitations-of-cxx.md
 msgid "Limitations of CXX"
-msgstr ""
+msgstr "حدود CXX"
 
 #: src/SUMMARY.md src/chromium/interoperability-with-cpp/error-handling.md
 msgid "CXX Error Handling"
-msgstr ""
+msgstr "معالجة الأخطاء في CXX (CXX Error Handling)"
 
 #: src/SUMMARY.md
 msgid "Error Handling: QR Example"
-msgstr ""
+msgstr "معالجة الأخطاء (Error Handling): مثال QR"
 
 #: src/SUMMARY.md
 msgid "Error Handling: PNG Example"
-msgstr ""
+msgstr "معالجة الأخطاء: مثال PNG"
 
 #: src/SUMMARY.md
 msgid "Using CXX in Chromium"
-msgstr ""
+msgstr "استخدام CXX في Chromium"
 
 #: src/SUMMARY.md src/chromium/adding-third-party-crates.md
 msgid "Adding Third Party Crates"
-msgstr ""
+msgstr "إضافة كريت الطرف الثالث (third party Crates)"
```

Comment:
here we don't translate crate while line 929: we use الحزم

Reply:
I think we need to get to a good translation for the keywords,

I started with this spreadsheet:
https://docs.google.com/spreadsheets/d/11Kn-C1xLqfIYRptC2ot0_efb_O-NTsQB8M1r_0focxo/edit?usp=sharing

Reply:
we need to agree their and changed later when we have a decision

Reply:
but for me, I am not sure if it should be `7ezam` or `krate` in arabic.

Reply:
The excel sheet is a nice idea, we can then meet and discuss

Reply:
A sheet is super for collaboration. Note that we also have a [glossary](https://google.github.io/comprehensive-rust/glossary.html) — this was partly created so that translations have a place to anchor their choices of translations.

The Markdown file has this comment, which is injected into the `.po` file:

https://github.com/google/comprehensive-rust/blob/be3a92a12a29b97baa195c9e37f28055bd3f18e4/src/glossary.md?plain=1#L27-L29

The idea is to make it possible to search for an English term ("crate") and then find the entry in the glossary, which tells you the translated term. This is just an idea, though, and I notice that other translations don't really follow it.

Reply:
Oh, if you find terms that are missing from the glossary, then please add them. It it by no means complete or exhaustive right now.

Reply:
Sure, thanks a lot Martin